packages/contrast-colors/wrapper.mjs

```diff
@@ -0,0 +1,13 @@
+import contrastColors from './index.cjs';
```

Comment:
@devongovett Parcel treats this like a static file you want to load on the client-side and returns a fingerprinted filename. Is there a way to make it treat it like `.js`?